src/raftstore/store/config.rs

```diff
@@ -27,6 +27,7 @@ const REGION_SPLIT_SIZE: u64 = 64 * 1024 * 1024;
 const REGION_MAX_SIZE: u64 = 80 * 1024 * 1024;
 const REGION_CHECK_DIFF: u64 = 8 * 1024 * 1024;
 const PD_HEARTBEAT_TICK_INTERVAL_MS: u64 = 60000;
+const DEAD_PEER_CHECK_TICK_INTERVAL: u64 = 180000;
```

Comment:
Anyway, on_dead_peer_check_result is unavoidable, because we need result from pd asynchronously. Then, having a on_dead_peer_check_tick symmetrically and a related ticker make the code readable...

I can change it if you persist.